pandas/core/frame.py

```diff
@@ -1697,19 +1697,42 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
 
         .. versionadded:: 0.21.0
 
+        This function writes the dataframe as a parquet file. You
+        can choose different parquet backends, and have the option
+        of compressing.
+
         Parameters
         ----------
         fname : str
-            string file path
+            String file path.
```

Comment:
IIRC we use lower for these? @jorisvandenbossche

Reply:
Here it is fine as start of the sentence and not being the exact type